src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -718,6 +777,40 @@ export default {
 		if (this.loanType === 'direct-loan') {
 			fireHotJarEvent('us_borrower_profile');
 		}
+
+		const challengeHeaderExpData = this.apollo.readFragment({
+			id: `Experiment:${CHALLENGE_HEADER_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+		this.enableChallengeHeader = challengeHeaderExpData?.version === 'b';
+
+		if (this.enableChallengeHeader) {
+			const teamPublicId = this.$route?.query?.team ?? '';
+			let teamId = null;
+			if (teamPublicId) {
+				const teamInfo = this.apollo.readQuery({ query: TeamInfoFromId, variables: { team_public_id: teamPublicId } }); // eslint-disable-line max-len
+				teamId = teamInfo?.community?.team?.id ?? null;
+			}
+			if (teamId) {
+				const teamData = this.apollo.readQuery({ query: TeamInfoFromId, variables: { team_id: teamId } });
+				this.teamData = teamData?.community?.team || {};
+
+				try {
+					const data = this.apollo.readQuery({
```

Comment:
This is the only `readyQuery` here within a try catch. I don't think `readQuery` is supposed to throw an exception, but I also know that we often wrap `readQuery` in a try catch. It would be worth investigating to see if an exception is thrown. If not, this try catch can be removed. If so, then the other `readQuery` usages in this method also need a try catch.